apps/application-system/api/migrations/20220112162352-pruneAt-and-prune-index.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(`
+      BEGIN;
+
+        CREATE INDEX pruned_and_pruneAt_idx ON application (prune_at) WHERE (pruned = FALSE AND prune_at IS NOT NULL);
```

Comment:
I expected that we'd need to index both properties, `(pruned, prune_at)` for the query planner to pick it up, since we filter on both of these properties. 

But maybe it's smart enough to know that the index is already filtered in a similar way. Please verify.